app/views/campaigns/toggle_archived.turbo_stream.erb

```diff
@@ -0,0 +1,3 @@
+<%= turbo_stream.update @campaign %>
+
+<%#= turbo_stream.update "campaign-counter", Campaign.where(archived: false).count  %>
```

Comment:
A theoretical example of the 'campaign counter' to show you could update multiple parts of a page with multiple `turbo_stream`s at once.